docs/src/developer-guide/code-conventions.md

```diff
@@ -11,3 +10,6 @@ The rationales for the specific rules in use can be found by looking to the
 project documentation for any given rule. If the rule is one of our own, see
 our own [rule documentation](https://eslint.org/docs/rules/) and otherwise, see
 the documentation of the plugin in which the rule can be found.
+
+If you need to make changes to a `package.json` file, please see the
+[package.json conventions](./package-json-conventions.md).
```

Comment:
```suggestion
[package.json conventions](https://deploy-preview-16206--docs-eslint.netlify.app/developer-guide/package-json-conventions).
```

Otherwise, on [the docs page](https://deploy-preview-16206--docs-eslint.netlify.app/developer-guide/code-conventions), it will redirect to a [404 URL](https://deploy-preview-16206--docs-eslint.netlify.app/developer-guide/package-json-conventions.md)